### DISCUSSION TITLE
Upgrade existing Cronicle

### DISCUSSION THREAD

**Discussion by ODA-AG** [jhuckaby/Cronicle#424] [Q&A] [answered]
You give great instructions from getting going with Cronicle, but what if we want to upgrade an existing Cronicle installs with the latest build?

**Comment by jhuckaby** [comment, marked as answer]
There's a section in the docs which covers this: https://github.com/jhuckaby/Cronicle#upgrading-cronicle

Sorry if it is hard to find.  It's buried down towards the bottom of the README.

Version 2 will have a much more structured documentation system.

**Comment by ODA-AG** [comment]
Thank you. That worked wonderfully.


Arnold Garcia | IT Manager | 503-460-0605 Ext. 100
On Demand Analysis · ***@***.***
Site.Work<https://www.site.work/> · ***@***.******@***.***>

**Comment by Mr-McBride** [comment]
Old thread, but it looks like the upgrade information has been removed.

**Reply by jhuckaby** [reply]
So sorry, a while back someone convinced me to split the doc into separate files, so now all the old doc links are broken.  Sigh.  Here's the new link, with my apologies: https://github.com/jhuckaby/Cronicle/blob/master/docs/CommandLine.md#upgrading-cronicle
